share/completions/hikari.fish

```diff
@@ -0,0 +1,8 @@
+set PROG 'hikari'
+
+complete -c $PROG -e 
```

Comment:
Erasing completions in a completion script isn't necessary.

This is either autoloaded the first time, or the user added some completions elsewhere, which is okay.